style-dictionary2/transforms/name_css_colors.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
```

Comment:
maybe we could create a library of transforms that get pulled into the old and new scripts?

Reply:
I can do that, good call